src/MoBi.Presentation/DTO/ExpressionParameterDTO.cs

```diff
@@ -0,0 +1,35 @@
+﻿using MoBi.Core.Helper;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Presentation.DTO;
+
+namespace MoBi.Presentation.DTO
+{
+   public class ExpressionParameterDTO : PathWithValueEntityDTO<ExpressionParameter>, IWithDisplayUnitDTO, IWithFormulaDTO
+   {
+
+      public ExpressionParameterDTO(ExpressionParameter expressionParameter) : base(expressionParameter)
+      {
+         ExpressionParameter = expressionParameter;
+         ContainerPath = expressionParameter.ContainerPath;
+         ExpressionParameter.PropertyChanged += underlyingObjectOnPropertyChanged;
```

Comment:
You will react to those changes twice. It's also done in the base class I am sure

Reply:
Thanks for catching that. I obviously implemented all this twice, then created the base class after